src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -80,6 +78,10 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord, SdkLimitO
             var attributeValueLengthLimit = sdkLimitOptions.AttributeValueLengthLimit;
             var attributeCountLimit = sdkLimitOptions.AttributeCountLimit ?? int.MaxValue;
 
+            /*
+            // Removing this temporarily for stable release
+            // https://github.com/open-telemetry/opentelemetry-dotnet/issues/4776
+            // https://github.com/open-telemetry/opentelemetry-dotnet/issues/3491
```

Comment:
We should also consider removing scope attributes for the initial stable release

https://github.com/open-telemetry/opentelemetry-dotnet/blob/79295b4e5489d952dd81ce2ccd12b54b94c31beb/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs#L154

There is an existing issue related to ILogger's scope attributes we have not yet resolved #4324.

Reply:
I think there's more value to exporting scope attributes as majority of the users who use scopes wouldn't run into the duplicate attributes issue and they would benefit from OTLPExporter exporting these attributes.

For #4324, I don't think the SDK should handle it. We can just put the onus on the user to not provide duplicate values or just have the backend systems deal with it.

Reply:
I haven't thought this through enough to provide a well-informed opinion. Though, if others share this stance and you're confident it's the right path, then I suggest closing #4324 as "by design". Perhaps this behavior is worth documenting?

Reply:
de-dup is going to be expensive, so we should not do it by default (we are not doing it for Metrics also). Post stable, we could consider (based on actual demand) to offer a Exporter specific feature to dedup, so the dedup cost is occurring in the dedicated thread, not in the main user threads.